models/DBClient.py

```diff
@@ -0,0 +1,28 @@
+import pymongo
+import json
+from bson import json_util
+class DBClient(object):
+    
+    #creates a collection for scouting data 
+    def __init__(self, ip, port):
+
+        self.client = pymongo.MongoClient(ip, port)
+        self.db = self.client['victiScout']
+        self.collection = self.db['scoutData']
+        self.data = []
+
+    def insertData(self, j):
+        #inserts into the collection (table)
+        self.collection.insert(j)
+
+    def getAllResults(self):
+        
+        #gets all scouting data in a cursor
+        cursor = self.collection.find()
+        
+        #converts cursor to json 
+        for doc in cursor:
```

Comment:
Instead of converting each element to json individually, add all of the docs to the list and *then* convert that to JSON.